adminer/content.md

```diff
@@ -30,6 +30,10 @@ Then point your web server to port 9000 of the container.
 
 Note: This exposes the FastCGI socket to the Internet. Make sure to add proper firewall rules or use a private Docker network instead to prevent a direct access.
 
+## %%STACK%%
+
+Run `docker stack deploy -c stack.yml %%REPO%%` (or `docker-compose -f stack.yml up`), wait for it to initialize completely, and visit `http://swarm-ip:8080`, `http://localhost:8080`, or `http://host-ip:8080` (as appropriate).
```

Comment:
I'm not really sure whether this really provides a value-add for Adminer: Usually you would use an existing database, instead of creating a new one to use.

The online demo using this `stack.yml` works fine, from what I've seen though.